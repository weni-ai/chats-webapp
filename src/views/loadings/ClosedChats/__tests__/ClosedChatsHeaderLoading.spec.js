import { mount } from '@vue/test-utils';
import ClosedChatsHeaderLoading from '../ClosedChatsHeader.vue';
import { beforeEach, describe, expect } from 'vitest';

const createWrapper = () => {
  return mount(ClosedChatsHeaderLoading);
};

describe('ClosedChatsHeaderLoading.vue', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = createWrapper();
  });

  it('to match snapshot', () => {
    expect(wrapper.html()).toMatchSnapshot();
  });
});
