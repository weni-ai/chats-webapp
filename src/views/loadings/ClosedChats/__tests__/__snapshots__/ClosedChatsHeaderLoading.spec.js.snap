// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`ClosedChatsHeaderLoading.vue > to match snapshot 1`] = `
"<header data-v-aa2d7ba8="" class="closed-chats-header-loading">
  <section data-v-aa2d7ba8="" class="closed-chats-header-loading__topbar">
    <div data-v-aa2d7ba8="" class="closed-chats-header-loading__topbar__breadcrumbs"><span data-v-aa2d7ba8=""><span class="unnnic-skeleton" style="background-color: rgb(232, 244, 244); animation: SkeletonLoading 1.5s ease-in-out infinite;"> ‌ </span></span><span data-v-aa2d7ba8=""><span class="unnnic-skeleton" style="background-color: rgb(232, 244, 244); animation: SkeletonLoading 1.5s ease-in-out infinite;"> ‌ </span></span><span data-v-aa2d7ba8=""><span class="unnnic-skeleton" style="background-color: rgb(232, 244, 244); animation: SkeletonLoading 1.5s ease-in-out infinite;"> ‌ </span></span></div><span data-v-aa2d7ba8=""><span class="unnnic-skeleton" style="background-color: rgb(232, 244, 244); animation: SkeletonLoading 1.5s ease-in-out infinite;"> ‌ </span></span>
  </section>
  <section data-v-aa2d7ba8="" class="closed-chats-header-loading__infos">
    <div data-v-aa2d7ba8=""><span class="unnnic-skeleton" style="background-color: rgb(232, 244, 244); animation: SkeletonLoading 1.5s ease-in-out infinite;"> ‌ </span></div>
    <section data-v-aa2d7ba8="" class="closed-chats-header-loading__infos__title">
      <div data-v-aa2d7ba8=""><span class="unnnic-skeleton" style="background-color: rgb(232, 244, 244); animation: SkeletonLoading 1.5s ease-in-out infinite;"> ‌ </span></div>
      <div data-v-aa2d7ba8=""><span class="unnnic-skeleton" style="background-color: rgb(232, 244, 244); animation: SkeletonLoading 1.5s ease-in-out infinite;"> ‌ </span></div>
    </section>
  </section>
</header>"
`;
